app_backend/balance.py

```diff
@@ -0,0 +1,55 @@
+from user import User
+
+
+class BalanceTracker(User):
+    def __init__(self, name, dooley_dollars, meal_swipes):
```

Comment:
I don't know if this matters but maybe you could do like:

`(self, name: string, dooley_dollars: float, meal_swipes: int)`

bc from this it seems like people can enter any value